index.js

```diff
@@ -25,6 +25,8 @@ function createSlimPreprocessor(config, logger) {
   if(typeof(config.slimrbOption) === 'string') {
     log.debug('Passing args to slim command (%s).', config.slimrbOption);
     slimCommandArgs.push(config.slimrbOption);
+  } else if(config && Array.isArray(config.slimrbOption)){
+    slimCommandArgs = config.slimrbOption;
```

Comment:
Can you add a debug logger here, like above?

`log.debug('Passing args to slim command (%s).', config.slimrbOption);
`

Reply:
ping

Reply:
Sorry about that. Pushed!